src/nvim/lua/executor.c

```diff
@@ -1268,14 +1268,14 @@ int nlua_source_using_linegetter(LineGetter fgetline, void *cookie, char *name)
 ///
 /// Used to call any lua callable passed from Lua into VimL
 ///
-/// @param[in]  lstate Lua State
 /// @param[in]  lua_cb Lua Callable
 /// @param[in]  argcount Count of typval arguments
 /// @param[in]  argvars Typval Arguments
 /// @param[out] rettv The return value from the called function.
-int typval_exec_lua_callable(lua_State *lstate, LuaCallable lua_cb, int argcount, typval_T *argvars,
-                             typval_T *rettv)
+int typval_exec_lua_callable(LuaCallable lua_cb, int argcount, typval_T *argvars, typval_T *rettv)
 {
+  lua_State *const lstate = global_lstate;
+
   LuaRef cb = lua_cb.func_ref;
```

Comment:
The changes in this file don't seem necessary and don't look to improve things?

Reply:
Oops yea these shouldn’t have been included